apps/andi/priv/repo/migrations/20200331174805_create_datasets.exs

```diff
@@ -7,6 +7,16 @@ defmodule Andi.Repo.Migrations.CreateDatasets do
       add :ingestedTime, :utc_datetime
     end
 
+    create table(:harvested_datasets, primary_key: false) do
```

Comment:
I actually don't know if you can edit an old migration file or if its customary to, but I feel like we've generally just made new files/migrations to add